libsolidity/formal/CHC.h

```diff
@@ -275,12 +270,24 @@ class CHC: public SMTEncoder
 	//@{
 	struct CHCVerificationTarget: VerificationTarget
 	{
-		smtutil::Expression errorId;
+		unsigned errorId;
+		ASTNode const * const errorNode;
+	};
+
+	struct CHCVerificationTargetPlaceholder
```

Comment:
Can you document what this struct is used for

Reply:
Documentation added.